src/java/fr/paris/lutece/plugins/document/web/DocumentResourceServlet.java

```diff
@@ -223,27 +223,35 @@ protected void processRequest( HttpServletRequest request, HttpServletResponse r
     /** Handles the HTTP <code>GET</code> method.
      * @param request servlet request
      * @param response servlet response
-     * @throws ServletException the servlet Exception
-     * @throws IOException the io exception
      */
     @Override
     protected void doGet( HttpServletRequest request, HttpServletResponse response )
-        throws ServletException, IOException
     {
-        processRequest( request, response );
+        try
+        {
+            processRequest( request, response );
+        }
+        catch( Exception e )
+        {
+            AppLogService.error( e.getMessage( ), e );
+        }
     }
 
     /** Handles the HTTP <code>POST</code> method.
      * @param request servlet request
      * @param response servlet response
-     * @throws ServletException the servlet Exception
-     * @throws IOException the io exception
      */
     @Override
     protected void doPost( HttpServletRequest request, HttpServletResponse response )
-        throws ServletException, IOException
     {
-        processRequest( request, response );
+        try
+        {
+            processRequest( request, response );
+        }
+        catch( Exception e )
+        {
+            AppLogService.error( e.getMessage( ), e );
```

Comment:
It is necessary to handle the exception in order to send a customized error message to the client.